install.go

```diff
@@ -212,7 +212,28 @@ func (gom *Gom) pullPrivate(srcdir string) (err error) {
 	defer os.Chdir(cwd)
 
 	fmt.Printf("fetching private repo %s\n", gom.name)
-	pullCmd := "git pull origin master"
+
+	branch := "master"
+ 	if has(gom.options, "branch") {
+ 		branch = gom.options["branch"].(string)
+ 	}
+ 
+ 	var vcs *vcsCmd
+ 	if isDir(filepath.Join(srcdir, ".git")) {
+ 		vcs = git
+ 	} else if isDir(filepath.Join(srcdir, ".hg")) {
+ 		vcs = hg
+ 	} else if isDir(filepath.Join(srcdir, ".bzr")) {
+ 		vcs = bzr
+ 	}
+ 	if vcs != nil {
+ 		err = vcs.Sync(srcdir, branch)
+ 		if err != nil {
+ 			return
+ 		}
+ 	}
+ 
+ 	pullCmd := "git pull origin " + branch
```

Comment:
Ah, right. I wonder why I did implement this. please delete old code for `git pull origin master`.

Reply:
Is this project no longer be under maintenance? This issue may be somewhat important, I think.

Reply:
> I mean my old code is wrong. So you don't need to keep my code at there. please remove it.

Hi, thanks for your attention. Lvyun was my coworker and we have tested these codes. We toke these changes in our private fork about one year ago and it works well. Without these codes, when updating private repos, gom will pull lateast remote master branch rather than remote target branch to the local target branch, it may lead to a merge confict(if master conficts which local target branch), nothing changes(if master is not updated), or an error(if master branch is not existed), which is not what we expected.
This feature is very important for us, but now for some seasons we have to delete our fork. So we are expecting your acception for this PR.

Reply:
Okay, Could you please make new PR from current master branch?

Reply:
Ok, and could you please just reopen this PR and accept it?
If there are any conflicts or some other problems, I will make a new PR tonight.

Reply:
I'll check this in later.

Reply:
Hi, how is it going?

Reply:
Sorry much. I'll check this tonight.